README.md

```diff
@@ -107,6 +107,15 @@ Load balancing and scaling out
 It is possible to run multiple versions of `gostatsd` behind a load balancer by having them
 send their metrics to another `gostatsd` backend which will then send to the final backends.
 
+Memory allocation for read buffers
+----------------------------------
+By default `gostatsd` will batch read multiple packets to optimise read performance. The amount of memory allocated
+for these read buffers is determined by the config options:
+
+    max-readers * receive-batch-size * 64KB (max packet size)
+
+The metric `avg_packets_in_batch` can be used to tune this memory usage if necessary.
```

Comment:
Maybe also mention `receive-batch-size` command line flag.

Reply:
I'll release today, and tweak when I tag (also update changelog)